### DISCUSSION TITLE
Integrations

### DISCUSSION THREAD

**Discussion by davidtabat** [breezefront/community#191] [3. Integrations]
Mollie https://github.com/mollie/magento2
Amasty GDPR https://amasty.com/gdpr-for-magento-2.html
Amasty Geoip https://amasty.com/geoip-for-magento-2.html
Amasty Product Labels https://amasty.com/product-labels-for-magento-2.html
Nvision Commerce365 https://n.vision/products/commerce-365-for-magento/
Magento QuickOrder and Smile quick order https://github.com/Smile-SA/magento2-module-elasticsuite-quick-order
Anowave extensions: https://www.anowave.com/marketplace/magento-2-extensions/magento-2-google-analytics-4-enhanced-ecommerce-tracking-gtm/ https://www.anowave.com/marketplace/magento-extensions/google-analytics-enhanced-ecommerce-web-tracking-for-magento/
Magictoolbox extensions: https://www.magictoolbox.com/magiczoomplus/modules/magento/ https://www.magictoolbox.com/magicscroll/modules/magento/
dotdigital extensions: https://github.com/dotmailer/dotmailer-magento2-extension https://github.com/dotmailer/dotmailer-magento2-extension-sms https://github.com/dotmailer/dotmailer-magento2-extension-chat

**Comment by dimdimich** [comment]
> Mollie https://github.com/mollie/magento2`

It's payment method so it should be working fine without any integration

> Amasty GDPR https://amasty.com/gdpr-for-magento-2.html

Please follow https://swissuplabs.com/blog/interested-in-amasty-modules-integration-with-breeze-check-this-information/ and contact Amasty.
You can also consider using https://swissuplabs.com/m2-gdpr-extension.html

> Amasty Geoip https://amasty.com/geoip-for-magento-2.html

Please follow https://swissuplabs.com/blog/interested-in-amasty-modules-integration-with-breeze-check-this-information/ and contact Amasty.

> Amasty Product Labels https://amasty.com/product-labels-for-magento-2.html

Please follow https://swissuplabs.com/blog/interested-in-amasty-modules-integration-with-breeze-check-this-information/ and contact Amasty.
You can also consider using https://swissuplabs.com/product-labels-extension-for-magento-2.html

> Nvision Commerce365 https://n.vision/products/commerce-365-for-magento/

As far as i can see this module is working only with backend so it doesn't require any integration

> Magento QuickOrder and Smile quick order https://github.com/Smile-SA/magento2-module-elasticsuite-quick-order

Smile is already integrated. Please check https://github.com/breezefront/module-breeze-smile-elasticsuite

> Anowave extensions: https://www.anowave.com/marketplace/magento-2-extensions/magento-2-google-analytics-4-enhanced-ecommerce-tracking-gtm/ https://www.anowave.com/marketplace/magento-extensions/google-analytics-enhanced-ecommerce-web-tracking-for-magento/

Please contact Anowave and ask them to add integration from their side. We are discussing this integration with their team and they said that it should be working almost without any problems

> Magictoolbox extensions: https://www.magictoolbox.com/magiczoomplus/modules/magento/ https://www.magictoolbox.com/magicscroll/modules/magento/

Our module already comes with zoom feature. Have you tried it?
